### PR TITLE
step sounds

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Improves step sounds.
IPCs will now play a servo moto sound effect when moving.
Riot suits and deathsquad armour now has a suit moving sound effect.

## Why It's Good For The Game

It sounds nice, gives some cool moving sounds to riot suits.

## Changelog
:cl:
add: Adds movement sounds to IPCs
add: Adds movement sounds to riot suits and deathsquad armour
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
In my opinion servostep would be too distracting for a sound that is constantly played such as footsteps.

**Comment by BitBite112** [comment]
What Belone said.

**Comment by hotimep** [comment]
maybe just a soft clank sound you know when walking on something metal

**Comment by theboy6545** [comment]
> Riot suits and deathsquad armour now has a suit moving sound effect

Psyops

File: code/datums/components/footstep.dm [hunk 1]
```diff
@@ -93,11 +93,13 @@
...
+			//Sound of wearing shoes always plays, special movement sound
+			// IE (server motors wont play bare footed.)
+			if(H.dna.species.special_step_sounds)
+				playsound(T, pick(H.dna.species.special_step_sounds), 50, TRUE)
+
+			else if((!H.shoes && !feetCover)) //are we NOT wearing shoes
+				playsound(T, pick(GLOB.barefootstep[T.barefootstep][1]),
+					GLOB.barefootstep[T.barefootstep][2] * v,
+					TRUE,
+					GLOB.barefootstep[T.barefootstep][3] + e)
```
Comment (utilbird) [inline code/datums/components/footstep.dm:105]: this does not bring joy

Comment (PowerfulBacon) [reply code/datums/components/footstep.dm:105]: I did not write this shit

**Pull request merged**